app/uk/gov/hmrc/tariffclassificationfrontend/views/not_authorized.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+
+@(resource: String = "Resource")(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Not required, but should we try to keep all the content under Messages?

Reply:
@pllaurado 
What do you mean?